### ISSUE TITLE
MetricAggregationCallback:   Different weights depending on the epoch

### ISSUE THREAD

**Issue by KirillTushin** [catalyst-team/catalyst#978]
Is it possible to set the dependence of the weights of loss functions over epochs?
For example, a loss for a model is a combination of two losses: Focal and Dice with weights
It is necessary that the weight of Focal decreases with epochs and the weight of Dice increases accordingly.
Can this be implemented using MetricAggregationCallback or using other methods, or do I need to implement such a Callback?

Example of feature :
```
MetricAggregationCallback(
    prefix="loss",
    mode="weighted_sum",
    metrics={
        "loss_focal": lambda epoch: 1/epoch,
        "loss_dice": lambda epoch: epoch/5,
    },
)
```

Thank you!

**Comment by Dokholyan** [comment]
Let's say we train a segmentation model, we want to teach it using a sum of Dice and BCE losses, after it, we would like to fine tuner model using Lovasz loss. It is better to do it smoothly. Example
```
def metric_aggregation(metrics: Dict[srt, torch.Tensor], runner: 'IRunner') -> torch.Tensor:
    """
    If epoch < 50, the weighted sum of base loss and finetuning loss will be used
    If epoch >=50, only finetuning loss will be used
    """
    epoch = runner.epoch
    
    # base training
    if epoch < 50:
        base_loss = 1 * metrics['loss_dice'] + 0.8 * metrics['loss_bce']
        finetuning_loss = metrics['loss_lovasz']
        loss = (1 - epoch / 50) * base_loss + (epoch / 50) * finetuning_loss
    # finetuning
    else:
        loss = metrics['loss_lovasz']
    return loss

MetricAggregationCallback(
    prefix="loss",
    mode=metric_aggregation,
    metrics=["loss_dice", 'loss_bce', 'loss_lovasz']
)
```
